daemon/containerfs_linux.go

```diff
@@ -118,6 +120,11 @@ func (daemon *Daemon) openContainerFS(container *container.Container) (_ *contai
 				if err := mount.Mount(m.Source, dest, "", opts); err != nil {
 					return err
 				}
+				if m.RecursivelyReadOnly {
+					if err := makeMountRRO(dest); err != nil {
```

Comment:
Is this something that should be provided by github.com/moby/sys/mount ? (either through a `mount.MakeRecursiveReadOnly()` or `mount.Mount()` accepting this as option?

Reply:
Not necessary IMHO, as the function just wraps a single `MountSetattr` call with the `EINTR` loop.